mobify/platform-scaffold
# Add tag build tools

There is sometimes a need to change and rebuild the tag for customer projects. That's not at all simple to do... but we have the tools to do it in the `mobify-tag` repo. I ported the build tools over to the scaffold.

 **JIRA**: N/A

## Changes
- Build a `package.json` for the `mobify-tag` repo that contains the single dependency and task
- Copy the `minify.js` script over, and modify it to operate in the different directory structure
- Add the instructions to the READMD

## Todos

## How to test-drive this PR
- `cd web/mobify-tag`
- Follow the README (`npm i`, make a change to `v8.html`, `npm run minify:v8`, check the changes are applied).
